aws/components/s3/setup.ftl

```diff
@@ -260,22 +260,26 @@
                     [/#if]
                     [#break]
 
+                [#case CDN_COMPONENT_TYPE]
                 [#case CDN_ROUTE_COMPONENT_TYPE ]
 
-                    [#local originPath = (linkTargetConfiguration.Solution.Origin.BasePath)?remove_ending("/") ]
+                    [#local cdnBaselineLinks = getBaselineLinks(linkTarget, [ "CDNOriginKey" ])]
+                    [#local cdnBaselineComponentIds = getBaselineComponentIds(cdnBaselineLinks)]
+                    [#local cdnCFAccessId  = getExistingReference(cdnBaselineComponentIds["CDNOriginKey"]!"", CANONICAL_ID_ATTRIBUTE_TYPE) ]
+
                     [#if linkDirection == "inbound" ]
                         [#local policyStatements +=
                             s3ReadPermission(
                                 s3Name,
-                                originPath,
+                                "",
```

Comment:
Will this not give CloudFront access to the entire ops bucket?

Reply:
This isn't for the Ops bucket and instead for linking to an S3 bucket.

Reply:
Thanks